code/modules/antagonists/traitor/syndicate_contract.dm

```diff
@@ -74,17 +74,15 @@
 				if (traitor_data.current_contract == src) 
 					traitor_data.current_contract = null
 
-			for(var/obj/item/W in M)
-				if (ishuman(M))
-					var/mob/living/carbon/human/H = M
-					if(W == H.w_uniform)
-						continue //So all they're left with are shoes and uniform.
-					if(W == H.shoes)
-						continue
+			if (!issilicon(M))
```

Comment:
How about you only strip carbons instead? This also breaks some other stuff, like the udders of cows.

The real fix would be to nullspace all the things that don't actually belong into contents but that's not gonna happen anytime soon.

Reply:
Oh... on second glance, it looks like you removed the code that actually strips people...

Reply:
wtf I actually did